docs/_mar-milestone_.md

```diff
@@ -45,3 +52,4 @@ is our wish list for this month:_
       values) #3607
 - [ ] Add support for "include" and "fields" to findById (REST API) #1721
```

Comment:
This is already supported. I added a test for `fields` as part of https://github.com/strongloop/loopback-next/pull/4745.